source/_posts/2018-10-01-release-79.markdown

```diff
@@ -54,7 +54,6 @@ Oh, and yes, there is also some cool new integrations. Support has been added fo
 - Add Logi Circle component, camera and sensor platform ([@evanjd] - [#16540]) ([logi_circle docs]) ([camera.logi_circle docs]) ([sensor.logi_circle docs]) (new-platform)
 - deCONZ cover support ([@Kane610] - [#16759]) ([cover docs]) ([deconz docs]) ([cover.deconz docs]) (new-platform)
 - GeoJSON platform ([@exxamalte] - [#16610]) ([geo_location docs]) (new-platform)
-- Add linky sensor ([@tiste] - [#16468]) ([sensor.linky docs]) (new-platform)
```

Comment:
Don't remove those, as they are still valid. Just remove the links to the documentation.

Same applies to all other changes.